test/multiverse/suites/activemerchant/Envfile

```diff
@@ -1,6 +1,6 @@
 if RUBY_VERSION >= '3.0.0'
   gemfile <<-RB
-    gem 'activemerchant', '~>1.121.0'
+    gem 'activemerchant' # latest
```

Comment:
I suggest going without this comment unless there's something specific that warrants it. It's redundantly describing default behavior and it's not repeated for all of the others in the same boat within this same heredoc block.